src/app/auth/onboarding/index.tsx

```diff
@@ -117,7 +119,7 @@ function OnboardingScreen() {
 
   return (
     <ScrollView style={styles.container}>
```

Comment:
We want to make this container a flex box, then add a `gap` of 16 so we can remove the marginBottom from the `UserStringInput`. Also add a `justifyContent: 'space-between'` so the update profile button goes to the bottom.